CHANGELOG.md

```diff
@@ -45,13 +45,14 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Bug Fixes
 
+* (x/bank) [#22543](https://github.com/cosmos/cosmos-sdk/pull/22543) Fix `DenomMetadata` rpc allow value with slashes.
```

Comment:
i think we should just link the PR that adds this code to the branch that holds this changelog (release/v0.53.x).

otherwise, this changelog becomes fairly hard to follow if someone wants to view the commit from a specific change. they'll be seeing the changes in the context of v52/v2 rather than v53, which makes it harder to decipher what code actually changed in relation to this branch